package.json

```diff
@@ -111,18 +111,29 @@
     "@types/react-router-dom": "^5.1.5",
     "@types/twilio-video": "^2.0.12",
     "@types/yup": "^0.29.3",
-    "@typescript-eslint/eslint-plugin": "^3.7.1",
-    "@typescript-eslint/parser": "^3.7.1",
+    "@typescript-eslint/eslint-plugin": "^4.8.1",
+    "@typescript-eslint/parser": "^4.8.1",
     "cypress": "^4.11.0",
     "cypress-file-upload": "^4.0.7",
+    "eslint": "^6.8.0",
+    "eslint-config-airbnb": "^18.2.1",
+    "eslint-config-airbnb-typescript": "^12.0.0",
+    "eslint-config-prettier": "^6.15.0",
     "eslint-config-react-app": "^5.2.1",
+    "eslint-loader": "^3.0.3",
+    "eslint-plugin-flowtype": "^4.6.0",
+    "eslint-plugin-import": "^2.22.1",
```

Comment:
- https://www.npmjs.com/package/eslint-plugin-import#rules

Big yes!

- https://github.com/benmosher/eslint-plugin-import/blob/HEAD/docs/rules/no-internal-modules.md
  - could be interesting
- https://github.com/benmosher/eslint-plugin-import/blob/HEAD/docs/rules/no-useless-path-segments.md
  - definitely
- https://github.com/benmosher/eslint-plugin-import/blob/HEAD/docs/rules/no-relative-parent-imports.md
  - probably not
- https://github.com/benmosher/eslint-plugin-import/blob/HEAD/docs/rules/no-named-as-default.md
  - unsure
- https://github.com/benmosher/eslint-plugin-import/blob/HEAD/docs/rules/no-named-as-default-member.md
  - unsure
- https://github.com/benmosher/eslint-plugin-import/blob/HEAD/docs/rules/no-deprecated.md
  - warn only at this stage
- https://github.com/benmosher/eslint-plugin-import/blob/HEAD/docs/rules/no-extraneous-dependencies.md
  - sounds good
- https://github.com/benmosher/eslint-plugin-import/blob/HEAD/docs/rules/exports-last.md
  - unsure how this will work with `export const foo = () =>` style (which I like)
- https://github.com/benmosher/eslint-plugin-import/blob/HEAD/docs/rules/extensions.md
  - yes, never use extensions (except for scss/svg/etc)
- https://github.com/benmosher/eslint-plugin-import/blob/HEAD/docs/rules/order.md
  - this is awesome! will help a lot. need to decide our canonical choices
- https://github.com/benmosher/eslint-plugin-import/blob/HEAD/docs/rules/newline-after-import.md
  - definitely
- https://github.com/benmosher/eslint-plugin-import/blob/HEAD/docs/rules/prefer-default-export.md
  - i don't like this
- https://github.com/benmosher/eslint-plugin-import/blob/HEAD/docs/rules/no-named-default.md
  - this is probably fine
- https://github.com/benmosher/eslint-plugin-import/blob/HEAD/docs/rules/no-default-export.md
  this would potentially be my preference, open to discussion around why we should/shouldn't though
- https://github.com/benmosher/eslint-plugin-import/blob/HEAD/docs/rules/group-exports.md
  - definitely